Makefile

```diff
@@ -12,6 +12,7 @@ LOGLEVEL ?= silent
 OSTYPE := $(shell uname -s | tr '[A-Z]' '[a-z]')
 COVTESTS ?= test
 GTEST_FILTER ?= "*"
+export PYTHON
```

Comment:
From what I read in the `Makefile` man it is only "exported" to subshells, no the caller.